apps/emqx_modules/README.md

```diff
@@ -0,0 +1,54 @@
+# EMQX Modules
+
+The application provide some minor functional modules that are not included in the MQTT
+protocol standard, including "Delayed Publish", "Topic Rewrite", "Topic Metrics" and "Telemetry".
+
+
+## Delayed Publish
+
+After enabling this module, messages sent by the user with the prefix
```

Comment:
This module provides a feature that allows for manual intervention in the publishing delay of messages in an MQTT system.
After the module is enabled, messages sent by the MQTT clients with the topic

> nit: only mqtt clinets? how to delay a message for gateway clinets?